pycode/memilio-epidata/memilio/epidata/getCommuterMobility.py

```diff
@@ -444,11 +598,32 @@ def get_commuter_data(setup_dict='',
     gd.write_dataframe(df_commuter_migration, directory, filename, file_format)
     gd.check_dir(os.path.join(directory.split('pydata')[0], 'mobility'))
     df_commuter_migration.to_csv(
-        directory.split('pydata')[0] + 'mobility/commuter_migration_scaled' +
+        directory.split('pydata')[0] + 'mobility/commuter_mobility' +
         '_20' + files[0].split('-20')[1][0: 2] + '.txt', sep=' ', index=False,
         header=False)
 
-    return df_commuter_migration
+    # Short end for cases where no center coordinates are given.
+    # this is especially important for 'test_epidata_get_vaccination_data'
+    if isinstance(center_coordinates, str):
```

Comment:
Please use default argument 'None' and check if this is 'None'

Reply:
added in latest commit